package.json

```diff
@@ -51,6 +52,8 @@
     "ember-qunit": "^4.4.1",
     "ember-resolver": "^5.0.1",
     "ember-route-action-helper": "^2.0.7",
+    "ember-sinon": "^4.0.0",
+    "ember-sinon-qunit": "^3.4.0",
```

Comment:
Added `ember-sinon-qunit`, which automatically installed `ember-sinon` to help with stubbing methods in the tests.